src/IndexNameMapper.php

```diff
@@ -65,7 +65,7 @@ public function getPureIndexName(string $fullIndexName): string
             return $matches[1];
         }
 
-        $prefixLength = \strlen($this->prefix);
+        $prefixLength = \strlen((string) $this->prefix);
```

Comment:
I think the following code is more semantic and avoid type juggling
```suggestion
        $prefixLength = $this->prefix ? \strlen($this->prefix) : 0;
```